WordPress/Sniffs/VIP/ValidatedSanitizedInputSniff.php

```diff
@@ -42,7 +42,7 @@ class ValidatedSanitizedInputSniff extends \WordPress\Sniffs\Security\ValidatedS
 	);
 
 	/**
-	 * Don't use.
+	 * Process the token and handle the deprecation notices.
```

Comment:
Please undo this change. This sniff was not touched by this PR. The description in this case is correct as the sniff has been moved (in a previous PR) and this version of the sniff should no longer be used.